test/lib/lbt/bundle/Builder.js

```diff
@@ -817,6 +817,198 @@ ${SOURCE_MAPPING_URL}=library-preload.js.map
 	]);
 });
 
+test.serial("integration: createBundle with depCache", async (t) => {
```

Comment:
Missing test for the AutoSplitter changes (also see [code coverage](https://coveralls.io/builds/63974262/source?filename=lib%2Flbt%2Fbundle%2FAutoSplitter.js#L78))

Reply:
Done